packaging/rpm/microshift.spec

```diff
@@ -18,8 +18,9 @@
 # SELinux specifics
 %global selinuxtype targeted
 %define selinux_policyver 3.14.3-67
-%define container_policyver 2.189.0
-%define container_policy_epoch 3
+# FIXME: Specified by make-rpm.sh
```

Comment:
I don't think ART's build system likes comments in .spec. @vfreex  PTAL

Reply:
CI didn't seem to like it anyway.